appshell/appshell_extensions_win.cpp

```diff
@@ -403,6 +411,77 @@ void CloseLiveBrowser(CefRefPtr<CefBrowser> browser, CefRefPtr<CefProcessMessage
     }
 }
 
+int32 OpenDocumentWithXDApp(ExtensionString path)
+{
+	ConvertToNativePath(path);
+	std::wstring args = ShellCommand + XDAppPublisherId + ShellCommandOptions + path;
+	DWORD result = (DWORD)ShellExecute(NULL, L"open", L"explorer.exe", args.c_str(), NULL, SW_SHOWDEFAULT);
```

Comment:
I googled/researched on how to launch UWP Applications from command line/programmatically. 
looks like It can be only launched using explorer.exe like below:
explorer.exe shell:appsFolder\<package_Id> /select,<document_path>

Reply:
will users of the system , who doesn't have admin rights can launch explorer this way?